spec/features/exhibits_show_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+describe 'Exhibits show page', type: :feature do
```

Comment:
```suggestion
RSpec.describe 'Exhibits show page', type: :feature do
```
Avoid the need for monkeypatching.